src/main/java/org/sonar/plugins/stash/StashPlugin.java

```diff
@@ -155,11 +157,18 @@ public void define(Context context) {
                           .options(SEVERITY_LIST_WITH_NONE).build(),
         PropertyDefinition.builder(STASH_INCLUDE_ANALYSIS_OVERVIEW)
                           .name("Include Analysis Overview Comment")
-                          .description("Create a comment to  the Pull Request providing a overview of the results")
+                          .description("Create a comment to the Pull Request providing a overview of the results")
                           .type(PropertyType.BOOLEAN)
                           .subCategory(CONFIG_PAGE_SUB_CATEGORY_STASH)
                           .onQualifiers(Qualifiers.PROJECT)
                           .defaultValue(Boolean.toString(DEFAULT_STASH_ANALYSIS_OVERVIEW)).build(),
+        PropertyDefinition.builder(STASH_INCLUDE_FILENAMES_IN_OVERVIEW)
+                          .name("Include Files in Overview")
+                          .description("Set to 0 to disable this feature or positive number to enable it. The number will be used as limit for how many files per issue will be shown in the overview.")

```

Comment:
Please change the order of the sentences. First say what it does, then explain how to disable it.